MonoGame.Framework/Graphics/SpriteFont.cs

```diff
@@ -117,7 +119,19 @@ public int GetHashCode(char b)
 		/// Gets or sets the character that will be substituted when a
 		/// given character is not included in the font.
 		/// </summary>
-		public char? DefaultCharacter { get; set; }
+		public char? DefaultCharacter
+        {
+            get { return _defaultCharacter; }
+            set
+            {
+                _defaultCharacter = value;
+                // Get the default glyph index here once.
+                if (_defaultCharacter.HasValue)
+                    TryGetGlyphIndex(_defaultCharacter.Value, out _defaultGlyphIndex);

```

Comment:
What happens here if the user assigns a character not in the font?

Reply:
TryGetGlyphIndex() will set _defaultGlyphIndex = -1, that's all. 
No Exception is thrown until GetGlyphIndexOrDefault() tries to draw the default character.
I tried to preserve the existing behavior and it pass the existing tests.
Perhaps XNA throws a differed message in that case, I 'll have to check ...

Reply:
OK, XNA throws an ArgumentException immediately when you set DefaultCharacter.
We have to add a test for this change too, I will leave this for later.